aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,170 @@
+import { Example } from '../../layouts';
+import { 
+  PageBackGroundExample,
+  PageExample, 
+  PageFullExample,
+  PageNarrowExample,
+  SinglePageContent,
+  PageWideExample 
+} from '../../examples';
+
+Page makes it easy to implement [page layouts](/templates/page-layouts) uniformly across HPE 
+applications. Two components, Page and PageContent, work together to ensure HPE theme-defined 
+padding, horizontal alignment, max-width, and responsive behaviors are applied in a consistent, 
+turn-key manner.
+
+<Example 
+  docs="https://v2.grommet.io/page?theme=hpe"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js"
+  showResponsiveControls={['fullScreen']}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  screenContainer={{scale: 0.85}}
+  width="100%"
+>
+  <PageExample />
+</Example>
+
+## Page and PageContent Working Together
+Page is the outer layout component and provides the desired layout context. 
+Page’s `kind` determines which theme-defined alignment, padding, and min/max-widths 
+are applied to its PageContent children.
+
+As a child of Page, PageContent consumes and adheres to the 
+layout styling provided by its parent Page.
+
+```
+<Page kind="wide">
+  <PageContent>
+
+    /* All contents within me will respect the 
+     * max-width, alignment, and padding specified 
+     * by Page's context. */
+     
+  </PageContent>
+  
+  /* Contents outside of PageContent are not bound by 
```

Comment:
should we add that everything should be in the `PageContent` by reading this teams may think hey if I want something to expand I'll just move it out of `PageContent` however that's not our recommendation correct?

Reply:
🤔 ... not sure.

Trying to explain the mechanics / provide the reader a mental model for how Page and PageContent relate. 

I'm not sure if that sort of caveat is valuable or not. @taysea thoughts?

Reply:
Hmmm.. I could go both ways on this. Since we are generally encouraging people to put page content inside of the PageContent, I'm slightly inclined to leave the "Contents outside ..." out for now.